commands/project.go

```diff
@@ -0,0 +1,93 @@
+package commands
+
+//handle the project logic for rhmap.
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/feedhenry/rhm/storage"
+	"github.com/urfave/cli"
+)
+
+//ProjectCmd constructs the required writer in order to send the response to the right place.
+type projectCmd struct {
+	out      io.Writer
+	in       io.Reader
+	response func(*http.Request) (*http.Response, error)
```

Comment:
In the login command I used the verb poster to indicate it was doing a post request. I think maybe doing the same here but calling it getter would make sense?